src/main/java/uk/gov/companieshouse/chipsrestinterfacesconsumer/service/impl/MessageProcessorServiceImpl.java

```diff
@@ -19,10 +22,18 @@
     @Autowired
     private ApplicationLogger logger;
 
+    @Autowired
+    private MessageProducer retryMessageProducer;
+
     @Override
-    public void processMessage(ChipsKafkaMessage message) {
+    public void processMessage(ChipsKafkaMessage message) throws ServiceException {
         Map<String, Object> logMap = Collections.singletonMap("Message", message);
         logger.info("About to send message to Chips", logMap);
-        chipsRestClient.sendToChips(message);
+        try {
+            chipsRestClient.sendToChips(message);
+        } catch (RestClientException restClientException) {
+            logger.error("Error seding message to chips will place on retry queue", restClientException, logMap);
```

Comment:
typo on 'seding'
I'd have a comma after Chips and change queue to topic

Reply:
you could also autowire in the retry topic name to print out the name of the topic the message will be sent - to help support when they read the logs

Reply:
I have added this in the logging in the retry producer itself